arch/arm/src/armv7-a/arm_vectors.S

```diff
@@ -40,9 +40,19 @@
  ****************************************************************************/
 
 	.data
+
+	.globl	g_irqtmp
+
 g_irqtmp:
+#ifdef CONFIG_SMP
+	.rept	CONFIG_SMP_NCPUS
+	.word	0		/* Saved lr */
+	.word	0		/* Saved spsr */
+	.endr
+#else
 	.word	0		/* Saved lr */
 	.word	0		/* Saved spsr */
+#endif
 
 g_undeftmp:
```

Comment:
how about g_undeftmp/g_aborttmp/g_fiqtmp? the same issue happen there.

Reply:
@xiaoxiang781216 

Thanks, I will extend the logic later.

Reply:
@xiaoxiang781216 

Done.